recipes/opsin/2.1.0/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
     name: opsin
     version: 2.1.0
 build:
-  number: 0
+  number: 1
```

Comment:
@johanneskoester As any other Java based tool in bioconda. Like https://github.com/bioconda/bioconda-recipes/blob/master/recipes/picard/meta.yaml
Do I miss something?